### PR TITLE
[fix #16] One click deployment of a private IOTA network

### LINKED ISSUE #16: One click deployment of a private IOTA network
1) Will have 4 AWS machines;
2) One IOTA "genesis" node;
     4 IOTA docker nodes,  1 <-> 2 <-> 3 <-> 4 <-> 1
     each of these 4 nodes will connect to the genesis node.

### PR DESCRIPTION
**Pull request by stplaydog**

### CODE REVIEW

File: scripts/iota_deploy/iota_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+IRI_JAR=iri-1.5.4.jar
+DAG1=13.229.64.41
```
Comment (wunder3605) [inline scripts/iota_deploy/iota_deploy.sh:4]: May pass ip address as parameter?

File: scripts/iota_deploy/iota_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# start the genesis node
+pssh  -i  -H ubuntu@${DAG1} -x "-oStrictHostKeyChecking=no  -i ~/gitlocal/dag.pem" 'kill $(ps aux | grep "java \-jar" | grep -v grep | awk "{print \$2}")'
+pssh  -i  -H ubuntu@${DAG1} -x "-oStrictHostKeyChecking=no  -i ~/gitlocal/dag.pem" 'java -jar ~/iota/run-time/iri-1.5.4.jar --testnet --testnet-no-coo-validation --snapshot=/home/ubuntu/iota/run-time/Snapshot.txt -p 14700 --max-peers 40 --remote &'
+
+# deploy in each machine
+pssh  -i  -h pssh_hosts -x "-oStrictHostKeyChecking=no  -i ~/gitlocal/dag.pem" 'mkdir -p ~/iota/run_time/data; touch ~/iota/run_time/neighbors'
+pssh  -i  -h pssh_hosts -x "-oStrictHostKeyChecking=no  -i ~/gitlocal/dag.pem" 'sudo rm -rf ~/iota/run_time/data/*;'
+pssh  -i  -h pssh_hosts -x "-oStrictHostKeyChecking=no  -i ~/gitlocal/dag.pem" 'sudo docker stop iota-node'
+pssh  -i  -h pssh_hosts -x "-oStrictHostKeyChecking=no  -i ~/gitlocal/dag.pem" 'sudo docker rm iota-node'
+pssh  -i  -h pssh_hosts -x "-oStrictHostKeyChecking=no  -i ~/gitlocal/dag.pem" 'echo udp://172.31.16.65:14600 > ~/iota/run_time/neighbors'
```
Comment (wunder3605) [inline scripts/iota_deploy/iota_deploy.sh:18]: Also can pass ip address  as parameter

**Review by wunder3605 — CHANGES_REQUESTED** [review]

File: scripts/iota_deploy/iota_deploy.sh (hunk 1 above)
Comment (xie4991) [reply scripts/iota_deploy/iota_deploy.sh:4]: Can you understand the value passing and address passing in address passing?

File: scripts/iota_deploy/iota_deploy.sh (hunk 2 above)
Comment (xie4991) [reply scripts/iota_deploy/iota_deploy.sh:18]: I understand this: Every request is different，ip and port number may be change, the address will not change.I don't know if it’s right

**Pull request merged**